packages/dashboard/src/API.tsx

```diff
@@ -31,7 +31,12 @@ export interface PluginServices {
         hasPassword(): Promise<boolean>
         changePassword(oldPassword: string, newPassword: string): Promise<void>
         setPassword(newPassword: string): Promise<void>
-        recoverWalletFromMnemonic(name: string, mnemonic: string, derivationPath?: string): Promise<string>
+        recoverWalletFromMnemonic(
+            name: string,
+            mnemonic: string,
+            derivationPath?: string,
+            isImported?: boolean,
```

Comment:
A wallet has only a set of mnemonic, can we recover wallets from another set of mnemonic words?

Reply:
No, but we can recovery wallet with private key and store json. And this is just a flag.